### PR TITLE
Personalized combat messages part 2

### PR DESCRIPTION
**Pull request by kingofkosmos**
## About The Pull Request
Adds more "personalized" combat messages for all participants in a fight: the attacker, the victim and the spectators. Continuing to do what the previous PR did in #46388 (previous was mostly unarmed combat).

Old combat messages (attacker & spectator get the same message):
old_scene

New combat messages (everyone gets their own message):
new_scene

PS. I'm not 100% sure how this should work in transforming.dm, _true_devil.dm, holo_effect.dm.

## Why It's Good For The Game
Easier to see what is happening and to whom.

## Changelog
:cl:
spellcheck: added personal versions for combat messages (e.g: "You whip John Smith!" instead of "Devin Spock whips John Smith!")
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -99,9 +99,12 @@
...
 /mob/living/carbon/send_item_attack_message(obj/item/I, mob/living/user, hit_area, obj/item/bodypart/hit_bodypart)
-	var/message_verb = "attacked"
-	if(length(I.attack_verb))
-		message_verb = "[pick(I.attack_verb)]"
+	var/message_verb_continuous = "attacks"
+	var/message_verb_simple = "attack"
+	if(length(I.attack_verb_continuous))
+		message_verb_continuous = "[pick(I.attack_verb_continuous)]"
+	if(length(I.attack_verb_simple))
+		message_verb_simple = "[pick(I.attack_verb_simple)]"
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/carbon_defense.dm:107]:
```suggestion
	var/message_verb_continuous = length(I.attack_verb_continuous) ? "[pick(I.attack_verb_continuous)]" : "attacks"
	var/message_verb_simple = length(I.attack_verb_simple) ? "[pick(I.attack_verb_simple)]" : "attack"
```
Ternary operators for syntactic sugar.

File: code/modules/antagonists/devil/true_devil/_true_devil.dm [hunk 2]
```diff
@@ -123,8 +123,8 @@
 	var/weakness = check_weakness(I, user)
 	apply_damage(I.force * weakness, I.damtype, def_zone)
 	var/message_verb = ""
-	if(I.attack_verb && length(I.attack_verb))
-		message_verb = "[pick(I.attack_verb)]"
+	if(I.attack_verb_continuous && length(I.attack_verb_continuous))
```
Comment (Rohesie) [inline code/modules/antagonists/devil/true_devil/_true_devil.dm:126]:
```suggestion
	if(length(I.attack_verb_continuous))
```
`length()` will return a falsey value without runtiming for falsey inputs

File: code/_onclick/item_attack.dm [hunk 3]
```diff
@@ -172,21 +172,26 @@
...
 /mob/living/proc/send_item_attack_message(obj/item/I, mob/living/user, hit_area, obj/item/bodypart/hit_bodypart)
-	var/message_verb = "attacked"
-	if(length(I.attack_verb))
-		message_verb = "[pick(I.attack_verb)]"
+	var/message_verb_continuous = "attacks"
+	var/message_verb_simple = "attack"
+	if(length(I.attack_verb_continuous))
+		message_verb_continuous = "[pick(I.attack_verb_continuous)]"
+	if(length(I.attack_verb_simple))
+		message_verb_simple = "[pick(I.attack_verb_simple)]"
```
Comment (Rohesie) [inline code/_onclick/item_attack.dm:180]:
```suggestion
	var/message_verb_continuous = length(I.attack_verb_continuous) ? "[pick(I.attack_verb_continuous)]" : "attacks"
	var/message_verb_simple = length(I.attack_verb_simple) ? "[pick(I.attack_verb_simple)]" : "attack"
```

**Comment by kingofkosmos** [comment]
Thank you @Rohesie!

**Pull request merged**